meta/contributor-handbook.md

```diff
@@ -0,0 +1,88 @@
+# Contributor Handbook
+
+This document lays out a basic guideline of how to write and contribute a new pattern.
+
+## Maturity Levels
+
+We defined four maturity levels for patterns: The higher a pattern's maturity level, the more strictly it was validated and the better is its writing.
+
+### In a Nutshell
+
+| \# | Name | Description | Usage | Time to merge |
+| ---- | ---- | ---- | ---- | ---- |
+| 1 | Initial | Contains an initial pattern idea or what we call a donut (a pattern with missing sections). | Not included in any publications | A few days / weeks |
+| 2 | Structured | Contains a complete pattern that it is not properly validated yet (e.g. because it only synthesizes an idea or the experiences from one instead of three organizations). | Included in the patterns catalogue but with a warning | A few days / weeks |
+| 3 | Validated | The pattern is validated (e.g. because three or more instances exists and are synthesized by the pattern) and its writing is of high quality. | Included in the patterns catalogue | One or two months |
+| 4 | Holy | The pattern is validated, its writing is of "text book" quality, and the community agrees it is responsible to use it as teaching material. | Included in the patterns catalogue and patterns book | As long as it takes |

```

Comment:
Strongly recommend that we not use the term Holy, both because some may consider this sacrilegious and because it seems to convey the sense that such a pattern cannot be revised, that it is perfect. That should never be the case.

How about: Established

Reply:
Thanks for this suggestion, @NewMexicoKid.

> some may consider this sacrilegious

I am really sorry. I was not intending to offend anyone. On the contrary, I want us to stay a community where every humam being feels welcomed - independently of religious beliefs, political views, gender identity, race, nationality, and any other factors.

> it seems to convey the sense that such a pattern cannot be revised, that it is perfect

What I wanted to convey is that we need to be extra careful because changes / revisions might come at a higher cost in this maturity level. I agree that `established` conveys this better. 

Let's wait another 24h for others to jump into the discussion. Otherwise, I'll change to `established`.

Reply:
```suggestion
| 4 | Established | The pattern is validated, its writing is of "text book" quality, and the community agrees it is responsible to use it as teaching material. | Included in the patterns catalogue and patterns book | As long as it takes |

```

Reply:
I adapted the documents to use the term `established` for the highest maturity level